packages/icon/tsconfig.build.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["src"],
+  "exclude": [
+    ".storybook",
+    "node_modules",
+    "dist",
+    "**/*.spec.*",
+    "**/*.story.*"
+  ],
+  "compilerOptions": {
+    "declaration": true,
+    "outDir": "dist",
+    "rootDir": "src"
```

Comment:
strict eventually?

Reply:
yeah. i'll follow up

Reply:
i checked and there isn't anything in the src. a couple things in stories we can update